deployment/HDI-Spark/Notebooks/README.md

```diff
@@ -0,0 +1,116 @@
+Deploy Spark .Net REPL to HDInsight Spark cluster with Notebook Experience
```

Comment:
the REPL isn't really a Spark .NET REPL, but a dotnet repl. Is it okay to brand it as a Spark .NET repl ?

Reply:
Changed